test/functional/testSettings.js

```diff
@@ -0,0 +1,739 @@
+// https://github.com/elastic/kibana-qa/blob/master/settings.md
+
+define(function (require) {
+
+  var registerSuite = require('intern!object');
+  var expect = require('intern/dojo/node!expect.js');
+  var ScenarioManager = require('intern/dojo/node!../fixtures/scenarioManager');
+  var fs = require('intern/dojo/node!fs');
+  var pollUntil = require('intern/dojo/node!leadfoot/helpers/pollUntil');
+  var Common = require('../support/pages/Common');
+  var SettingsPage = require('../support/pages/SettingsPage');
+  var HeaderPage = require('../support/pages/HeaderPage');
+  var config = require('intern').config;
+  var url = require('intern/dojo/node!url');
+  var _ = require('intern/dojo/node!lodash');
+
+
+  registerSuite(function () {
+    var common;
+    var settingsPage;
+    var headerPage;
+    var scenarioManager;
+
+    var expectedAlertText = 'Are you sure you want to remove this index pattern?';
+    return {
+      // on setup, we create an settingsPage instance
+      // that we will use for all the tests
+      setup: function () {
+        common = new Common(this.remote);
+        settingsPage = new SettingsPage(this.remote);
+        headerPage = new HeaderPage(this.remote);
+        scenarioManager = new ScenarioManager(url.format(config.elasticsearch));
+      },
+
+      beforeEach: function () {
+        var remote = this.remote;
```

Comment:
instead of having this before each test, can this be done once in the setup method?